src/ai/susi/server/api/aaa/GetApiKeys.java

```diff
@@ -93,8 +93,8 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
             if (authentication.getIdentity() != null) {
                 ClientIdentity identity = authentication.getIdentity();
                 String userId = identity.getUuid();
-                JSONObject userKeys = apiKeys.getJSONObject("user");
-                configKeys = userKeys.has(userId) ? userKeys.getJSONObject(userId) : null;
+                JSONObject userKeys = userId != null && apiKeys.has("user") ? apiKeys.getJSONObject("user") : null;
+                configKeys = userKeys != null && userKeys.has(userId) ? userKeys.getJSONObject(userId) : null;
```

Comment:
In case there is no entry for user, we need to create the entry. That case is not handled.